### DISCUSSION TITLE
Adding disqus comments to Gatsby blog

### DISCUSSION THREAD

**Discussion by sashirajc** [LekoArts/gatsby-themes#897] [Questions]
Hello!

I'm trying to add https://www.gatsbyjs.com/plugins/gatsby-plugin-disqus/ to a new gatsby blog. But I'm not sure where to update the JSX code since I haven't been able to identify the JSX template files (Screenshot attached). Any help is appreciated. Thank you.
<img width="1050" alt="Screenshot 2022-05-17 at 11 58 27 AM" src="https://user-images.githubusercontent.com/13371143/168725869-2cb1f01f-428f-4c87-afcd-9946e9e44a11.png">

**Comment by LekoArts** [comment]
You can shadow the `PostFooter` (https://github.com/LekoArts/gatsby-themes/blob/main/themes/gatsby-theme-minimal-blog/src/components/post-footer.tsx), as the name suggests it's shown under the post. Place your Disqus JSX there.
